com.unity.robotics.urdf-importer/Tests/Runtime/UrdfComponents/UrdfJoints/UrdfJointTests.cs

```diff
@@ -0,0 +1,303 @@
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using RosSharp.Urdf;
+
+public class TestUrdfJoint : UrdfJoint
+{
+    public override JointTypes JointType => throw new System.NotImplementedException();
+
+    public static Vector3 Axis(RosSharp.Urdf.Joint.Axis axis) => GetAxis(axis);
+    public static Vector3 DefaultAxis() => GetDefaultAxis();
+    public static RosSharp.Urdf.Joint.Axis AxisData(Vector3 axis) => GetAxisData(axis);
+    public void Dynamics(RosSharp.Urdf.Joint.Dynamics dynamics) => SetDynamics(dynamics);
+}
+
+public class UrdfJointTests
+{
+#if UNITY_2020_1_OR_NEWER
```

Comment:
For my understanding, why are the tests only for Unity 2020_1 or newer?

Reply:
ArticulationBody is a new Unity feature that is only available in 2020.1 or newer versions. The old versions only have the joint APIs.

Reply:
my understanding is that with the scripting define symbol these tests only run if the editor is 2020.1 or newer. Is it correct? Do we need to add edit mode tests for older version editor?

Reply:
Yeah, I also thought about it when writing a test, but I guess it might be a bigger change and needs further discussions (created the [AIRO-688](https://jira.unity3d.com/browse/AIRO-688) to follow up). 

Yes, you are right that we are only testing 2020.1 or newer versions. On the other hand, we are advertising the articulation body, which becomes available in PhysX 4 and uses Featherstone algorithms that are commonly used for robotics simulation, so my idea was to remove the support of the rigidbody joints (e.g. floating, planar, continuous, and hinge, etc.). 

In addition, I think it might also worth keeping these types of rigidbody joints for a completely different purpose. As you may know, we are recommending people split the robot into multiple articulation body chains if there are more than 64 joints. The separated articulation bodies then need to be connected by configurable joints or rigidbody joints. But it is still not clear to me how to do it in the URDF-Importer.

Reply:
Yeah, a follow-up discussion with the team sounds good.